minichatgpt/backend/server.py

```diff
@@ -49,4 +50,4 @@ def run_chatbot_web():
 
 
 if __name__ == "__main__":
-    bot.run(host='127.0.0.1', port=8001)
+    bot.run(host='0.0.0.0', port=8001)
```

Comment:
@eruigu I  changed the address to `0.0.0.0` so it can be reachable in a docker. I think we need to add a boolean to check whether it's prod (ie in a docker) then use the 0.0.0.0 address otherwise use `127.0.0.1`. I'll add this before we merge so it's usable locally on the terminal

Reply:
I've added a check for whether its running in production,  and an env file to get variables in local development